tools/alloy-core-rs-example/src/main.rs

```diff
@@ -0,0 +1,83 @@
+use std::str::FromStr;
+use alloy::{
+    hex,
+    network::{EthereumWallet, ReceiptResponse, TransactionBuilder},
+    primitives::U256,
+    providers::{Provider, ProviderBuilder},
+    rpc::types::TransactionRequest,
+    signers::local::PrivateKeySigner,
+    sol,
+};
+use eyre::Result;
+use std::env;
+use dotenv::dotenv;
+
+sol! {
```

Comment:
Bytecode is already embedded in `main`, where is this macro being used?

Reply:
If this is not used, we should remove it.

Reply:
@acuarica 
This sol! macro generates a struct that corresponds to the Solidity contract defined in the JSON artifact. We are using it on this [line](https://github.com/hashgraph/hedera-json-rpc-relay/pull/2773/files#diff-b78e3f872da383914a721d1256e7222ffe1180b3ef95af16f9024d6c58519f5eR44) . I believe that in the current version of this PR, the bytecode is no longer embedded in the main.rs file. However, even if it were, this struct could still be used to interact with the corresponding Smart Contract.

What concerns me more is that I don't see any actual usage of either ReceiptResponse or TransactionBuilder. We should remove them from the imports, @mateuszm-arianelabs .

Reply:
Make sense, thanks @arianejasuwienas for the clarification.

Reply:
Yes, we need this code to correspond to the Solidity contract. I have removed unused imports and reformatted the code.